src/sentry/management/commands/backfill_eventstream.py

```diff
@@ -54,7 +55,7 @@ def _attach_related(_events):
             for event in _events:
                 event.project = projects[event.project_id]
                 event.group = groups[event.group_id]
-            Event.objects.bind_nodes(_events, "data")
+            eventstore.bind_nodes(_events, "data")
```

Comment:
What's the reason to use eventstore in some cases ?

Reply:
Two use cases
- the main one: it takes advantage of nodestore multi_get so if we want to fetch an entire page of nodestore objects it is more efficient
- when binding raw event (1 case), we need to use this now since that doesn't inherit from EventCommon